### DISCUSSION TITLE
Should mermaid edges to and from subgraphs be supported?

### DISCUSSION THREAD

**Discussion by loewenstein** [community/community#40742] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I wanted to use the mermaid integration and failed with the following:

```
graph LR
  start --> next
  next --> sub
  subgraph sub
    sub1 --> sub2
  end
  sub --> End
```

When I paste the same to the mermain.live editor, this renders sucessfully. See [this example](https://mermaid.live/edit#pako:eNo1j7EOwjAMRH8l8twOMGZggg0WWLO4tUsjNUmVOhKo6r-TpGWy_XzW-VboAzFoMP4dcR7V_Wm8UotgFNW2F-X5I4WUWsGSuqpI3X5wzJWc_opzQezpUFZ88wQNOI4OLWXHtSwNyMiODejcEg-YJjH5mS1LMUl4fX0PWmLiBtJMKHy1mI0d6AGnJVMmKyE-9hQ1zPYDaG5FPA)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Loewenstein! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
